src/domain/app/i18n/en.json

```diff
@@ -399,6 +400,7 @@
     "warnings": {
       "capacityInWaitingList": "Registration for this event is still possible, but there are only {{count}} seat left in the queue.",
       "capacityInWaitingList_other": "Registration for this event is still possible, but there are only {{count}} seats left in the queue.",
+      "capacityInWaitingListNoLimit": "Registration for this event is still possible, but there are only seat left in the queue.",
```

Comment:
Should probably be in plural 🤔 `Registration for this event is still possible, but there are only seats left in the queue.`

Reply:
Good catch. Fixed